### ISSUE TITLE
Using with tensorflow

### ISSUE THREAD

**Issue by Ajaz-Ahmad** [oval-group/smooth-topk#11]
Hi Team,

Thanks for sharing your wonderful work. I am working with Tensorflow and wanted to use your loss function. Would it be possible to use this or will I need to implement? Any idea how can I proceed with it.
Thank you.

**Comment by lberrada** [comment]
Hi, thank you for your interest. We don't support any tensorflow unfortunately, you would need to reimplement the algorithm for that.
